lib/Settings/About.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:url_launcher/url_launcher.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+
+class About extends StatelessWidget {
+  final controller = ScrollController();
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: Text('About'),
+      ),
```

Comment:
I've implemented a `BaseAppBar` that looks a little something like the snippet below - can you pull from `master` and use this component instead?

```
      appBar: BaseAppBar(
        title: Text('Install TTS'),
        appBar: AppBar(),
      ),
```

Reply:
Done!